### PR TITLE
Update 3rd-party-licenses.md

### LINKED ISSUE #12763: [Improvement][Docs]: Versions of 3rd party libraries
### Improvement description

Would it make sense to add the versions for the 3rd party libraries somewhere?  
I would propose it in this file https://github.com/pimcore/pimcore/blob/11.x/3rd-party-licenses.md

### PR DESCRIPTION
**Pull request by mysliwietzflorian**
- refer to `PIMCORE_MODIFICATIONS.md` to see the versions of 3rd party libraries

  

## Changes in this pull request  

Resolves #12763

## Additional info  

I though some cross-referencing might help 😉

### CODE REVIEW

File: 3rd-party-licenses.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ The Pimcore project is licensed under the terms of the [GPLv3](LICENSE.md).
 However, Pimcore includes several third-party Open-Source libraries, 
 which are licensed under their own respective Open-Source licenses.
 
+The specific verions of a library are documented in
+[PIMCORE_MODIFICATIONS.md](bundles/AdminBundle/Resources/public/js/lib/PIMCORE_MODIFICATIONS.md).
```
Comment (dvesh3) [inline 3rd-party-licenses.md:10]:
```suggestion
[PIMCORE_MODIFICATIONS.md](https://github.com/pimcore/pimcore/tree/11.x/bundles/AdminBundle/Resources/public/js/lib/PIMCORE_MODIFICATIONS.md).
```

Comment (mysliwietzflorian) [reply 3rd-party-licenses.md:10]: @dvesh3 Is this actually a good idea? We would need to update that for every future branching change.

The way I wrote it should work for future changes where the path has not changed and forked repos, etc. WDYT?

Comment (dvesh3) [reply 3rd-party-licenses.md:10]: yes, you are right. I misunderstood it with docs so should be fine like this.

**Comment by dvesh3** [comment]
@mysliwietzflorian thanks!

**Pull request merged**